recipe/.env

```diff
@@ -0,0 +1,4 @@
+DB_CONNECTION=mongodb+srv://dood1:6iCdoaNy551UMgMx@main-cluster-vjjaz.mongodb.net/RecipeDB?retryWrites=true&w=majority
+PORT = 3000
```

Comment:
Go to the drive under dev/ENV update dev.env and use that instead (note your changes in Changelog).

Also update this and the package.json to folder /config/dev.env. Its better code practice than committing your DB username and password to a public repository